.release-version

```diff
@@ -1 +1 @@
-3.37.0
+3.37.0-uat1
```

Comment:
set as 3.37.1 for a minor change like this, don't use suffixes like -uat1
the github actions will automatically run and create a 3.37.1 develop release you can deploy into UAT